src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -3855,6 +4099,30 @@ components:
             transaction:
               $ref: "#/components/schemas/CommittedTransactionInfo"
 
+    TransactionSubintentDetails:
+      type: object
+      required:
+        - subintent_hash
+      properties:
+        subintent_hash:
+          $ref: "#/components/schemas/HashBech32mString"
+        manifest_instructions:
+          description: |
+            A text-representation of a subintent transaction manifest.
+            This field will be present only for user transactions and when explicitly opted-in using `manifest_instructions` flag.
+          type: string
+          nullable: true
```

Comment:
Does this need to be `nullable`? Or is non-required enough?

Reply:
no, removed.